css/style.css

```diff
@@ -0,0 +1,1002 @@
+  font-size: 44px;
+  margin-bottom: 0;
+}
+
+.location-paragraph {
+  color: #272a31;
+  font-family: "Lato", sans-serif;
+  font-size: 20px;
+  line-height: 35px;
+  padding-top: 20px;
+}
+
+.past-summit {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+  text-align: center;
+  height: 80vh;
+  padding-left: 24px;
+  padding-right: 24px;
+  padding-bottom: 70px;
+}
+
+.summit {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.past-summit-heading {
+  color: #272a31;
+  font-size: 44px;
+  margin-bottom: 0;
+}
+
+.past {
+  color: #272a31;
+  font-size: 22px;
+}
+
+.past-summit-container {
+  display: grid;
+  grid-template-columns: auto auto;
+  gap: 50px;
+}
+
+.past-summit-card1 {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center center;
+  text-decoration: none;
+  height: 250px;
+  padding-left: 5px;
+  padding-right: 5px;
+  mix-blend-mode: multiply;
+  background-image:
+    linear-gradient(
+      to bottom,
+      rgba(255, 28, 28, 0.52),
+      rgba(255, 28, 28, 0.52)
+    ),
+    url(../Images/summit1.jpg);
+}
+
+.past-summit-card2 {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center center;
+  text-decoration: none;
+  height: 250px;
+  padding-left: 5px;
+  padding-right: 5px;
+  mix-blend-mode: multiply;
+  background-image:
+    linear-gradient(
+      to bottom,
+      rgba(255, 28, 28, 0.52),
+      rgba(255, 28, 28, 0.52)
+    ),
+    url(../Images/summit3.jpg);
+}
+
+.past-summit-card1:hover {
+  background-image: url(../Images/summit1.jpg);
+}
+
+.past-summit-card2:hover {
+  background-image: url(../Images/summit3.jpg);
+}
+
+.card-paragraph {
+  font-size: 15px;
+  color: #fff;
+}
+
+.sponsors {
+  display: none;
+}
+
+.foot {
+  display: none;
+}
+
+.sponsors-about {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background: #272a31;
+  width: 100%;
+  padding: 20px 0;
+}
+
+.sponsors-about h2 {
+  color: #d3d3d3;
+  margin-bottom: 10px;
+}
+
+.bronze-about {
+  display: grid;
+  grid-template-columns: auto auto auto auto;
+  gap: 20px;
+  padding-top: 50px;
+  padding-left: 24px;
+  padding-right: 24px;
+}
+
+.bronze-about li {
+  display: flex;
+  background: #474747;
+  width: 100%;
+}
+
+.bronze-about li img {
+  width: 100%;
+}
+
+.foot-about {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: auto;
+  gap: 20px;
+  padding-bottom: 30px;
+  width: 100%;
+  padding-top: 30px;
+}
+
+.logo-about {
+  display: flex;
+  width: 180px;
+  align-items: center;
+}
+
+.copyright-about {
+  color: #272a31;
+  font-family: "Lato", sans-serif;
+}
+
+@media screen and (min-width: 768px) {
+  .social {
+    display: flex;
+    justify-content: end;
+    gap: 50px;
```

Comment:
- [ ] Kindly consider lowering the value of `gap` property because the requirement is to make this top navbar aligned to the right side as in the Behance design. A value of `gap: 20px` would be better.